src/win32.mak

```diff
@@ -256,7 +256,7 @@ auto-tester-build: $G dmd checkwhitespace $(DMDFRONTENDEXE)
 dmd: $G reldmd
 
 $(GEN)\build.exe: build.d $(HOST_DMD_PATH)
-	$(HOST_DC) -of$@ -debug build.d
+	$(HOST_DC) -m$(MODEL) -of$@ -debug build.d
```

Comment:
Since the build.exe is model-specific now.  It probably needs 2 names now.  Is there a reason why this needs to match the same model as the dmd exe we are building?

Reply:
It should be fine without multiple versions, it produces the same output. I don't like using -m32 cause I've experienced numerous problems that are fixed by not using it (one of which is not printing output correctly). So anyone building a -m32 DMD can continue to do so without VS.